mycli/main.py

```diff
@@ -825,14 +853,23 @@ def sanitize(col):
                 cur, headers, format_name='vertical' if expanded else None,
                 column_types=column_types,
                 **output_kwargs)
-            first_line = formatted[:formatted.find('\n')]
+
+            if isinstance(formatted, (str, text_type)):
```

Comment:
I think we can drop `str` now that the unicode problems in CLI Helpers were fixed.